apps/mobile/src/components/widgets/tokens/tokens.mocks.tsx

```diff
@@ -0,0 +1,121 @@
+import { t } from '@lingui/macro';
```

Comment:
I'm using this mock file to think through what we actually need to do here. 

In the extension, we have separate files with `BtcCryptoAssetItem`, `StacksCryptoAssetItem` and the code can be a bit hard to follow. 

That could be something we need to do based on having to have a `Loader` per token. 

I will try to create a serialiser that gathers the data then creates a `tokens` array of objects with all the data needed and map over that.